cluster/service/cluster_service_blackbox_test.go

```diff
@@ -533,6 +533,138 @@ func (s *ClusterServiceTestSuite) TestLoadForAuth() {
 		})
 	})
 }
+func (s *ClusterServiceTestSuite) TestFindByURL() {
+
+	s.T().Run("ok", func(t *testing.T) {
+
+		for _, saName := range []string{"fabric8-oso-proxy", "fabric8-tenant", "fabric8-jenkins-idler", "fabric8-jenkins-proxy", "fabric8-auth"} {
+			t.Run(saName, func(t *testing.T) {
+				// given
+				c := newTestCluster()
+				err := s.Application.ClusterService().CreateOrSaveCluster(context.Background(), c)
+				require.NoError(t, err)
+				sa := &authtestsupport.Identity{
+					Username: saName,
+					ID:       uuid.NewV4(),
+				}
+				ctx, err := authtestsupport.EmbedServiceAccountTokenInContext(context.Background(), sa)
+				require.NoError(t, err)
+				// when
+				result, err := s.Application.ClusterService().FindByURL(ctx, c.URL)
```

Comment:
Would be nice to test that both urls with a trailing slash and without it work fine.

Reply:
done in e4586bc